plugins/power/gpm-common.c

```diff
@@ -743,6 +745,8 @@ backlight_set_percentage (GnomeRRScreen *rr_screen,
         if (max < 0)
                 return ret;
         discrete = PERCENTAGE_TO_ABS (0, max, *value);
+        /* For not as dark as 0 */
+        discrete = MAX (discrete, PERCENTAGE_TO_ABS (0, max, 1));
```

Comment:
Nitpick: Since `*value` is a percentage already, it would be clearer to apply the clamping one line earlier, before `discrete` is calculated. i.e.
```
discrete = PERCENTAGE_TO_ABS (0, max, MAX (*value, 1));
```

Reply:
I have updated the [upstream's MR](https://gitlab.gnome.org/GNOME/gnome-settings-daemon/merge_requests/17).  Thanks
Should we cherry pick again?

Reply:
I also updated to the new code here.

Reply:
Thanks, merging.